### PR TITLE
chore: update dependencies to the latest version

### PR DESCRIPTION
**Pull request by ersachin3112**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case
update dependencies to the latest version

### Breaking Changes
No

### Additional Info
No

### CODE REVIEW

**Review by snitin315 — DISMISSED** [review]

File: test/__snapshots__/loader.test.js.snap [hunk 1]
```diff
@@ -2,18 +2,7 @@
 
 exports[`loader should do nothing with built-in CSS support: errors 1`] = `Array []`;
 
-exports[`loader should do nothing with built-in CSS support: warnings 1`] = `
-Array [
-  "ModuleWarning: Module Warning (from ../../node_modules/css-loader/dist/cjs.js):
-You can't use \`experiments.css\` (\`experiments.futureDefaults\` enable built-in CSS support by default) and \`css-loader\` together, please set \`experiments.css\` to \`false\` or set \`{ type: \\"javascript/auto\\" }\` for rules with \`css-loader\` in your webpack config (now css-loader does nothing).",
```
Comment (snitin315) [inline test/__snapshots__/loader.test.js.snap:8]: Can you check why was this removed?

**Comment by alexander-akait** [comment]
Close due https://github.com/webpack-contrib/style-loader/pull/616
